stream_alert/alert_processor/outputs/pagerduty.py

```diff
@@ -401,6 +441,11 @@ def dispatch(self, **kwargs):
         if rule_context:
             rule_context = rule_context.get(self.__service__, {})
 
+        # Use the priority provided in the context, use it or the incident will be low priority
+        incident_priority = {}
+        if rule_context:
```

Comment:
one last suggestion - don't even create the `incident_priority` here or do the `if rule_context` check here.. do this all in the `_priority_verify` and return `dict()` immediately if the rule_context == None